CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
+## [3.1.3] - 2023-07-13
```

Comment:
This will be released tomorrow (most likely), so I set the date to 2023-07-13 although today is the 12th